### DISCUSSION TITLE
Why do greater/less than & not equal symbols automatically default to a subscript for the following number?

### DISCUSSION THREAD

**Discussion by AFlowOfCode** [artisticat1/obsidian-latex-suite#337] [Q&A] [answered]
For example, if I write >= it gets converted to `\geq` automatically. If I then type something immediately after, it gets converted into `\get_{3}`.

 ![geq_sub](https://github.com/user-attachments/assets/91696c8c-dbb9-4197-9661-a5de16be4c0f)

This happens with <=, !=, and directly using the slash directives. Perhaps it's my mathematical ignorance here (and apologies if so), but adding subscript directly afterward can't possibly be the most common use case for these symbols. 

I didn't want to file a bug in case this is on purpose, but my question is - why? And how can I disable this? It reduces my velocity having to go back and undo the underscore & brackets every time. I looked in the settings & the snippets and there seems to be no way to edit this. 

Edit: thought I should clarify that I'm aware if you press space after the snippet replacement then it works without a subscript. I'd just like to know why the default is to assume subscript should follow these symbols.

Thank you!

**Comment by AFlowOfCode** [comment]
There are in fact many symbols where this happens. Writing `->` converts to `\to` and then immediately typing a number results in `\to_{3}` which is subscript. 

The same happens for `ln` -> `\ln_{4}`. The natural log case in particular makes no sense, since "ln" already implies a base of `e` and if you want a different base, you should use "log".

Given that there has been no response to this discussion, I'm assuming nobody knows any logical justification and this indeed a bug. I'll wait a bit to see if anyone responds, but at this point I'm fairly convinced this should be filed as a bug report.

**Comment by bdm-k** [comment]
Hello,

I just wanted to let you know that your question is entirely valid. Latex Suite should change its default settings to something more sensible. Thank you for bringing this up!

**Reply by AFlowOfCode** [reply]
Thanks for the response. Let me know if you want me to make an issue for reference purposes.
Latex Suite is awesome, happy to help if I can!

**Comment by Amberg22** [comment]
A fix for this would be to rewrite those codes that do this, like cdot, to:

`{trigger: "cdot", replacement: "\\,\\cdot\\, ${0: }", options: "mA"},`

And remove cdot from Snippet variables. Didnt have to do it for the other ones for it to work. For those that do work, just write

`{trigger: "->", replacement: "\\to ${0: }", options: "mA"},`

**Comment by hieunguyen1212** [comment]
You can use the snippet below and add `le`, `ge` into the list of more symbols.
```
{trigger: "\\\\(${MORE_SYMBOLS})", replacement: "\\[[0]] ", options: "rmA"}
```

**Comment by superle3** [comment, marked as answer]
from https://github.com/artisticat1/obsidian-latex-suite/discussions/630#discussioncomment-18309250 you can do
```ts
	{
	  trigger: /(\\?)([A-Za-z]+)(\d)/,
	  replacement: (match) => {
	   const isMacro = match[1] === "\\";
	   const digit = match[3];
	   if (!isMacro) {
		   const variable = match[2];
		   return variable + "_{" + digit + "}";
	   }
	   const greek = require("latex-suite").snippetVariables["${GREEK}"];
	   const greek_pattern = new RegExp("^(?:" + greek + ")$");
	   const macro_name = match[2];
	   if (greek_pattern.test(macro_name)) {
			return `\\${macro_name}_{${digit}}`;
	   } else {
			return `\\${macro_name} ${digit}`;
	   }
	  },
	  options: "rmA",
	  priority: -1,
	  excludedMacros: ["pu", "ce"]
	}
```
to automatically add spaces for digits after a macro if its not a greek symbol

which is now in the default snippets. (snippets are not updated, you have to do that manually)
